firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Util/cobs.txt

```diff
@@ -18,10 +18,14 @@ Scenario: The Util encode_cobs function correctly encodes buffers
     THEN("The encode_cobs function reports ok status")
     THEN("The encoded buffer is as expected '\x03\x11\x22\x02\x33'")
 
-  WHEN("The cobs::encode function is called on a buffer that contains these bytes '0x11 0x22 0x00 0x33' and a non-empty output_buffer is passed")
+  WHEN("The cobs::encode function is called on a buffer that contains these bytes '0x11 0x22 0x00 0x33' and a output_buffer filled with 2-bytes is passed")
```

Comment:
When you say that the output_buffer is "filled", is it actually full (which would imply that its capacity is 2 bytes? Or is it just supposed to be partially filled? This applies to the other WHENs changed in this commit.

Reply:
It is partially filled, so a 254-byte buffer is filled with 2-bytes.
How do you suggest I frame this WHEN?